modules/ztp-du-cluster-config-reference.adoc

```diff
@@ -0,0 +1,92 @@
+= Recommended cluster configurations to run vDU applications
+
+Clusters running virtualized distributed unit (vDU) applications require a highly tuned and optimized configuration. The following information describes the various elements that you require to support vDU workloads in {product-title} {product-version} clusters.
+
+[NOTE]
+====
+You manage your custom site configuration data in a Git repository accessible from the hub cluster that is defined as a source repository for Argo CD. You extract the baseline source custom resources (CRs) from the zero touch provisioning (ZTP) container.
+====
+
+== Recommended cluster MachineConfig installation CRs
+
+The following `MachineConfig` installation CRs configure the cluster host:
+
+.MachineConfig installation CRs
+[cols=2*, width="80%", options="header"]
+|====
+|CR filename
+|Description
+
+|`02-workload-partitioning.yaml`
+|Configures workload partitioning for the cluster.
+
+|`MachineConfigSctp.yaml`
+|Loads the SCTP kernel module. This `MachineConfig` CR is optional and can be omitted if you do not require this kernel module.
+
+|`MachineConfigContainerMountNS.yaml`
+|Configures the container mount namespace and kubelet conf.
+
+|`MachineConfigAcceleratedStartup.yaml`
+|Configures accelerated startup for the cluster.
+
+|`06-kdump-master.yaml`, `06-kdump-worker.yaml`
+|Configures kdump for the cluster.
+|====
+
+== Recommended cluster Operators
+
+The following Operators are required for clusters running vDU applications and are installed when the cluster is provisioned with the ZTP pipeline:
+
+. Node Tuning Operator (NTO). NTO packages functionality that was previously delivered with the Performance Addon Operator, which is now a part of NTO.
+. PTP Operator
+. SR-IOV Network Operator
+. Red Hat OpenShift Logging Operator
+. Local Storage Operator
+
+== Recommended cluster kernel configuration
+
+Always use the latest supported realtime kernel version in your cluster. You should also ensure that the following configurations are applied in the cluster:
+
+. Ensure the following `additionalKernelArgs` are set in the cluster performance profile:
++
+[source,yaml]
+----
+spec:
+  additionalKernelArgs:
+  - "idle=poll"
+  - "rcupdate.rcu_normal_after_boot=0"
+  - "efi=runtime"
+----
+
+. Ensure that the `performance-patch` profile in the `Tuned` CR configures the correct CPU isolation set that matches the `isolated` CPU set in the related `PerformanceProfile` CR, for example:
++
+[source,yaml]
+----
+spec:
+  profile:
+    - name: performance-patch
+      # The 'include' line must match the associated PerformanceProfile name
+      # And the cmdline_crash CPU set must match the 'isolated' set in the associated PerformanceProfile
+      data: |
+        [main]
+        summary=Configuration changes profile inherited from performance created tuned
+        include=openshift-node-performance-openshift-node-performance-profile
+        [bootloader]
+        cmdline_crash=nohz_full=2-51,54-103 <1>
```

Comment:
This does not match with the config in performance profile. 
Either change performanceprofile configs to 0-1,52-53, or change this config to 3-39,43-79. 
@imiller0 do we prefer to configure 6 or 4 reserved cpus?

Reply:
Prefer 4 to match up with wording elsewhere. 4 is the minimum necessary reserved with additional possibly needed based on the users workload.

Reply:
I have updated examples based on the following CPU allocations:

```
  cpu:
    isolated: 0-1,52-53
    reserved: 2-4,40-42
```